### ISSUE TITLE
猪耳朵识别软件1

### ISSUE THREAD

**Issue by de7ga** [de7ga/ze#1]
python
# 安装必要库
!pip install tensorflow opencv-python numpy matplotlib
!pip install yolov5  # 如果选择YOLOv5
2  数据准备与预处理
python
import cv2
import numpy as np
from sklearn model_selection import train_test_split
 
def load_and_preprocess_images(image_paths, labels, target_size=(224, 224)):
    images = []
    for path in image_paths:
        img = cv2 imread(path)
        img = cv2 cvtColor(img, cv2 COLOR_BGR2RGB)
        img = cv2 resize(img, target_size)
        images append(img)
    return np array(images)/255 0, np array(labels)
 
# 示例数据加载（实际应用中需要真实数据集）
# X, y = load_and_preprocess_images(['pig_ear1 jpg', 'pig_ear2 jpg'], [1, 1])
